app/src/main/res/layout/fragment_add_edit_submission.xml

```diff
@@ -28,12 +28,20 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
+                <ImageView
+                    android:id="@+id/submission_add_edit_back_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="25dp"
+                    android:src="@drawable/ic_arrow_back_black_36dp"
+                    >
+                </ImageView>
```

Comment:
minor personal preference - you could close these two ImageViews with /> instead of the closing tag